typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -90,15 +91,31 @@ export function deserializeMarketData(data: Buffer): MarketData {
     restingOrderBeet
   );
 
-  // TODO: Respect price-time ordering
-  const bids = [...bidsUnsorted].sort((a, b) =>
-    sign(toBN(b[0].priceInTicks).sub(toBN(a[0].priceInTicks)))
-  );
+  // Sort bids in descending order of price, and ascending order of sequence number
+  const bids = [...bidsUnsorted].sort((a, b) => {
+    const priceComparison = sign(
+      toBN(b[0].priceInTicks).sub(toBN(a[0].priceInTicks))
+    );
+    if (priceComparison !== 0) {
+      return priceComparison;
+    }
+    return sign(
```

Comment:
Please double-check this @jarry-xiao

Reply:
I think this is correct. If b.price < a.price, then a is "smaller". If they have the same price then the one with the "smaller" sequence number is chosen.